package.json

```diff
@@ -30,7 +30,8 @@
     "Sami Koskimäki <sami.koskimaki@vincit.com> (https://github.com/koskimas)",
     "Mikael Lepistö <mikael.lepisto@vincit.com> (https://github.com/elhigu)",
     "Matthew McEachen <matthew-objection@photostructure.com> (https://github.com/mceachen)",
-    "Jürg Lehni <juerg@scratchdisk.com> (https://github.com/lehni)"
+    "Jürg Lehni <juerg@scratchdisk.com> (https://github.com/lehni)",
+    "Igor Savin <kibertoad@gmail.com> (https://github.com/kibertoad)"
```

Comment:
(this addition was approved by @koskimas but it's too insignificant to warrant a separate PR)